src/components/tools/details/ToolDetails.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+
+import { useTranslation } from "i18n";
+
+import GridLabelValue from "components/utils/GridLabelValue";
+import GridLoading from "components/utils/GridLoading";
+import ErrorTypographyWithDialog from "components/utils/error/ErrorTypographyWithDialog";
+
+import { Grid, Paper, Typography } from "@material-ui/core";
+
+export default function ToolDetails(props) {
+    const { baseDebugId, tool, isInfoFetching, infoFetchError } = props;
+    const { t } = useTranslation("tools");
+
+    if (isInfoFetching) {
+        return <GridLoading rows={10} baseId={baseDebugId} />;
+    }
+
+    if (infoFetchError) {
+        return (
+            <ErrorTypographyWithDialog
+                errorObject={infoFetchError}
+                errorMessage={t("toolInfoError")}
+            />
+        );
+    }
+
+    if (!tool && !isInfoFetching && !infoFetchError) {
+        return null;
+    }
+
+    return (
+        <Paper id={baseDebugId} style={{ padding: 5 }}>
```

Comment:
Looking at the screenshots, how does it look without the wrapping `Paper`?  It looks a little cramped to me with the paper added, not sure what others think.

Reply:
Removing `Paper` makes it better. So I will go ahead and do that.
![image](https://user-images.githubusercontent.com/1250790/102406115-20196680-3fb8-11eb-8703-caa06384ddd7.png)